flytekit/core/base_task.py

```diff
@@ -477,6 +479,8 @@ def __init__(
                 execution of the task. Supplied as a dictionary of key/value pairs
             disable_deck (bool): (deprecated) If true, this task will not output deck html file
             enable_deck (bool): If true, this task will output deck html file
+            decks (Tuple[DeckField]): Tuple of decks to be
```

Comment:
I think naming this `deck_fields` is better to distinguish it from all the actual decks:

```suggestion
            deck_fields (Tuple[DeckField]): Tuple of decks to be
```